### PR TITLE
Release mcan/0.5.0

### PR DESCRIPTION
**Pull request by Ironedde**
# Release 0.5.0
### Added
- Add safe way to shutdown the bus when actively transmitting/receiving (#45)
- Add method to finalize configuration into initialization mode (#47)

### Changed
- *Breaking* Update the register mappings with svd2rust 0.30.2 and form 0.10.0 (#46)

## Thank you!

Thank you for your contribution.
Please make sure that your submission includes the following:

### Must

- [x] The code compiles without `errors` or `warnings`.
- [x] All tests pass and in the best case you also added new tests.
- [x] `cargo +stable fmt` was run.
- [x] `cargo +stable clippy` yields no `warnings`.
- [x] Your changes were added to the `CHANGELOG.md` in the proper section.
- [x] You add a description of your work to this PR.
- [x] You added proper docs (in code, rustdoc and README.md) for your
      newly added features and code.

### CODE REVIEW

**Comment by Ironedde** [comment]
## Todo after merge
- [ ] Add tag mcan/0.5.0 to the master branch
- [ ] Publish to crates.io

**Review by vcrn — DISMISSED** [review]
Looks good! :+1:

File: mcan/CHANGELOG.md [hunk 1]
```diff
@@ -3,10 +3,16 @@
 Tagging in git follows a pattern: `mcan/<version>`.
 
 ## [Unreleased]
-- *Breaking* Update the register mappings with svd2rust 0.30.2 and form 0.10.0 (#46)
+
+## [0.5.0] - 2023-10-24
```
Comment (epontan) [inline mcan/CHANGELOG.md:7]: Wrong date :smile:

**Review by epontan — CHANGES_REQUESTED** [review]
Looks good, just update the date

File: mcan/CHANGELOG.md (hunk 1 above)
Comment (Ironedde) [reply mcan/CHANGELOG.md:7]: Haha, copy paste always fails :rofl:

Comment (Ironedde) [reply mcan/CHANGELOG.md]: Fixed :+1:

**Review by epontan — APPROVED** [review]
Nice! LGTM :+1:

**Pull request merged**